osu.Game/Screens/Edit/Verify/VerifyScreen.cs

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Linq;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Beatmaps;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Rulesets;
+using osu.Game.Rulesets.Edit;
+using osu.Game.Rulesets.Edit.Checks.Components;
+using osuTK;
+
+namespace osu.Game.Screens.Edit.Verify
+{
+    public class VerifyScreen : EditorScreen
+    {
+        private Ruleset ruleset;
+        private static IBeatmapVerifier beatmapVerifier;
```

Comment:
as a heads up, `static` is banned in this project for use cases like this.

Reply:
right, I was intending to ask about this but forgot. looking at how you solved it answers my questions though